Content/CodeInjection.md

```diff
@@ -48,7 +48,7 @@ The `EntrypointC` function ensures that it will be executed only once, using a s
 
 ##### InitInject
 
-The `InitInject` function checks if the process is running natively (i.e. 32-bit on a x86 system or 64-bit in a x64 system), or if its running under wow64 (that is a 32-bit process on a 64-bit system) and selects eider the native ntdll base address or the one of the wow64 ntdll. On windows prior to 8 that address was located in `KUSER_SHARED_DATA::Wow64SharedInformation` structure, but not on later windows. Sandboxie used the driver to record the address of the wow64 ntdll during image loading and `InitInject` queried the driver for it. Since version 5.44 however its driver independent, the loader code uses `NtQueryVirtualMemory` to find the image base address and saves it into the `ntdll_wow64_base` field of the data block.
+The `InitInject` function checks if the process is running natively (i.e. 32-bit on a x86 system or 64-bit in a x64 system), or if its running under wow64 (that is a 32-bit process on a 64-bit system) and selects either the native ntdll base address or the one of the wow64 ntdll. On windows prior to 8 that address was located in `KUSER_SHARED_DATA::Wow64SharedInformation` structure, but not on later windows. Sandboxie used the driver to record the address of the wow64 ntdll during image loading and `InitInject` queried the driver for it. Since version 5.44 however its driver independent, the loader code uses `NtQueryVirtualMemory` to find the image base address and saves it into the `ntdll_wow64_base` field of the data block.

```

Comment:
```suggestion
The `InitInject` function checks if the process is running natively (i.e. 32-bit on a x86 system or 64-bit in a x64 system), or if it's running under wow64 (that is a 32-bit process on a 64-bit system) and selects either the native ntdll base address or the one of the wow64 ntdll. On Windows versions prior to 8, that address was located in `KUSER_SHARED_DATA::Wow64SharedInformation` structure, but not on later versions. Sandboxie used the driver to record the address of the wow64 ntdll during image loading and `InitInject` queried the driver for it. Since version 5.44, however, it's driver independent, the loader code uses `NtQueryVirtualMemory` to find the image base address and saves it into the `ntdll_wow64_base` field of the data block.
```